server/src/server.ts

```diff
@@ -6527,553 +6359,130 @@ Email verified! You can close this tab or hit the back button.
   }
 
   const getUser = User.getUser;
+  const getComments = Comment.getComments;
+  const _getCommentsForModerationList = Comment._getCommentsForModerationList;
+  const _getCommentsList = Comment._getCommentsList;
+  const getNumberOfCommentsRemaining = Comment.getNumberOfCommentsRemaining;
```

Comment:
Where did these come from?

Reply:
Not sure why git borked the diff so bad, but this was already here, so we're fine.

Reply:
something must have shifted in some strange way in a rebase or similar git foo. These popped up out of a big deleted block below and they don't belong in the code anymore. Troubling... going to have to double check for other loose code chunks.